idris-mode.el

```diff
@@ -90,10 +102,10 @@ Invokes `idris-mode-hook'."
   (set (make-local-variable 'parse-sexp-lookup-properties) t)
   (set (make-local-variable 'syntax-propertize-function) 'idris-syntax-propertize-function)
 
-  ; REPL completion for Idris source
+                                        ; REPL completion for Idris source
```

Comment:
No. My elisp auto indenter went a bit amok it would seem. I'll change it
back when I wake up tomorrow.

/David (from phone)
Den 18 jan 2015 04:03 skrev "Hannes Mehnert" <notifications@github.com>:

> In idris-mode.el
> <https://github.com/idris-hackers/idris-mode/pull/294#discussion-diff-23132807>
> :
>
> > @@ -90,10 +102,10 @@ Invokes `idris-mode-hook'."
> >    (set (make-local-variable 'parse-sexp-lookup-properties) t)
> >    (set (make-local-variable 'syntax-propertize-function) 'idris-syntax-propertize-function)
> >
> > -  ; REPL completion for Idris source
> > +                                        ; REPL completion for Idris source
>
> any reason for this and following whitespace changes?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idris-hackers/idris-mode/pull/294/files#r23132807>.
>